doc/classes/EditorInterface.xml

```diff
@@ -15,60 +17,70 @@
 			<argument index="0" name="resource" type="Resource">
 			</argument>
 			<description>
+				Edits the given [Resource].
 			</description>
 		</method>
 		<method name="get_base_control">
 			<return type="Control">
 			</return>
 			<description>
+				Returns the base [Control].
 			</description>
 		</method>
 		<method name="get_edited_scene_root">
 			<return type="Node">
 			</return>
 			<description>
+				Returns the edited scene's root [Node].
 			</description>
 		</method>
 		<method name="get_editor_settings">
 			<return type="EditorSettings">
 			</return>
 			<description>
+				Returns the [EditorSettings].
 			</description>
 		</method>
 		<method name="get_editor_viewport">
 			<return type="Control">
 			</return>
 			<description>
+				Returns the editor viewport.
```

Comment:
I'm not sure about this one. The return type says `Control` - what object is actually being returned?

Reply:
It returns `Viewport` or `NULL`, just tested this.